enatega-multivendor-app/src/assets/SVG/Bicycle.js

```diff
@@ -2,10 +2,13 @@ import * as React from "react"
 import Svg, { G, Path, Defs, ClipPath } from "react-native-svg"
 
 function Bicycle(props) {
+  const { size } = props;
```

Comment:
if you dont provide size, then this will break with error "cant destructure properties of undefined ..... ", so instead use props?.size

Reply:
done